### PR TITLE
 Components Directory And  U.I

### PR DESCRIPTION
**Pull request by Cosmosha**
The following requirement and feature was implemented;

- No Linter Error
- Add React components directory 
- Create Calculator.js
-  Add Calculator U.I

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <button type="button"> 1 </button>
+        <button type="button"> 2 </button>
+        <button type="button"> 3 </button>
+        <button type="button" className="operators"> + </button>
+        <button type="button" className="span-two"> 0 </button>
+        <button type="button"> . </button>
+        <button type="button" className="operators"> = </button>
+      </div>
+    </div>
+  );
```
Comment (TadesseDev) [inline src/components/calculator.js]: - [ ] Dear @Cosmosha, As part of the main requirement for this project, your `Calculator.js` should have at least 1 child component. Please consider taking some logic into a separate component and including it in your `Calculator` component.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Cosmosha 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7


## Highlights
 - [x]  Correct GitFlow  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
- [ ] Good job adding the MIT file, The problem is you need to update its content in regard to your name and project title. Please replace the `[YOUR NAME]` part with your name, `[APP TYPE]` with your app type, and the [YEAR] with the current year. 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (Cosmosha) [reply src/components/calculator.js]: Hello @TadesseDev, 
thank you for the feedback, however please can you simplify or help with me a clear direction on what I should do?
I am somehow confused.

Comment (TadesseDev) [reply src/components/calculator.js]: Dear @Cosmosha, In react we can render multiple components inside a single component. In your case, you can have a separate component for your buttons and import your button to the `calculator` component and render it as a child component. Please refer to [this blog](https://towardsdev.com/parent-to-child-component-communication-in-react-ea5b673de6ce) for more, it has more explanation on `why` and `how`.

Comment (Cosmosha) [reply src/components/calculator.js]: Thank you so much @TadesseDev.

**Review by uwadonat — APPROVED** [review]
## Hi @Cosmosha,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Good commit messages ✔️
- Correct workflow was used ✔
approval1
️

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**